java/rapidoid/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM maven:3.6-jdk-8 as maven
```

Comment:
As I understand, `maven` build this on `java` **8**, but this run  on `jre` **11**

Reply:
Yes, that is true (also on the pom.xml the source is specified as java 8), but there was no specific intent.
This means we will not be using any language feature of java 11 but the runtime environment will be more current. What do you think we should do?

Reply:
It's a little bit confusing :stuck_out_tongue: 
I suggest to use the same runtime either for build and for run